release_notes/46-release-notes.adoc

```diff
@@ -0,0 +1,320 @@
+
+:FeatureName: compliance reporting
+include::snippets/technology-preview.adoc[]
+
+Compliance reporting is available as a Technology Preview for all OpenShift clusters running Compliance Operator version 1.6 or higher. It allows customers to easily access the compliance results of a given scan schedule in a CSV file.
+
+Compliance reporting provides the following options:
+
+* Generating the report on demand directly by using the {product-title-short} web portal or the API
+* Sending the report periodically by email every time a scan is scheduled
+* Creating email notifiers when creating a scan configuration as a destination to send on-demand reports
+* Generating an on-demand report for a given scan configuration, which will be sent to any notifiers configured on that scan configuration
+
+For more information, see xref:../operating/manage-compliance/scheduling-compliance-scans-and-assessing-profile-compliance.adoc#scheduling-compliance-scans-and-assessing-profile-compliance[Scheduling compliance scans and assessing profile compliance].
+
+[id="network-graph-external-IP_{context}"]
+== External IP address information available in network graph
+
+The network graph now provides additional insights into external entities. With this update, you can view specific IP addresses associated with these external connections, offering a more comprehensive understanding of network activity.
+
+For more information, see
+// PR not merged yet - link won't work until it is: `../operating/visualizing-external-entities.adoc#visualizing-external-entities[Visualizing external entities]`.
+
+[id="microsoft-sentinel-notifier_{context}"]
+=== Microsoft Sentinel notifier added
+
+{product-title-short} has added a Microsoft Sentinel notifier to send alerts and audit logs to Azure Log Analytics Workspace. For more information, see xref:../integration/integrating-with-microsoft-sentinel-notifier.adoc#integrating-with-microsoft-sentinel-notifier[Integrating with Microsoft Sentinel notifier].
+
+[id="s3-clone-support-backup_{context}"]
+=== Support for backups using non-AWS S3 compatible providers
+
+{product-title-short} has added a new external backup integration for non-AWS S3 compatible providers. For more information, see xref:../integration/integrate-with-s3-api-compatible-services.adoc#integrate-with-s3-api-compatible-services[Integrating with S3 API compatible services].
+
+[id="violations-platform-components_{context}"]
+=== Platform components displayed in the Violations page
+
+No release note text in Jira?
+
+[id="vulnerability-management-updates_{context}"]
+=== Vulnerability Management page updates
+
+CVE published date now included::
+
+{product-title-short}, in vulnerability management, now reports the CVE published date. The CVE published date is the date when the CVE entry was first published in the National Vulnerability Database (NVD) or by the vendor in vendor specific security data feeds when those are available.
```

Comment:
I think it does, but I want to ensure it is clear we prefer the date from the vendor and then only use NVD when that info is missing